### PR TITLE
DEVDOCS-4669: [update] Price Lists, rate limits for simultaneous in-flight requests

### PR DESCRIPTION
**Pull request by bc-tgomez**
# [DEVDOCS-4669]

## What changed?
* Added rate limit detail and information.

## Anything else?
Related PR: https://github.com/bigcommerce/api-specs/pull/1088 

[DEVDOCS-4669]: https://bigcommercecloud.atlassian.net/browse/DEVDOCS-4669?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: reference/price_lists.v3.yml [hunk 1]
```diff
@@ -850,7 +850,12 @@ paths:
       tags:
         - Price Lists Records
       summary: Get All Price List Records
-      description: Returns a list of *Price List Records* associated with a *Price List*.
+      description: |-
+        Returns a list of *Price List Records* associated with a *Price List*.
+
+        **Notes**
+        * Up to 10 simultaneous GET requests are supported with this API. Running more than the allowed concurrent requests in parallel on the same store will result in a `429` status error and your additional requests will fail.
```
Comment (slsriehl) [inline reference/price_lists.v3.yml:857]:
```suggestion
        * Supports up to 10 simultaneous GET requests. Running more than the allowed number of requests concurrently on the same store will result in a `429` status error and your additional requests will fail.
```
`concurrent requests in parallel` is redundant.

File: reference/price_lists.v3.yml [hunk 2]
```diff
@@ -1761,7 +1766,12 @@ paths:
       tags:
         - Price Lists Records
       summary: Get Price Records by Variant
-      description: Returns *Price List Records* using the variant ID. Will also contain currency records.
+        description: |-
+        Returns *Price List Records* using the variant ID. Will also contain currency records.
+        
+        **Notes**
+        * Up to 50 simultaneous GET requests are supported with this API. Running more than the allowed concurrent requests in parallel on the same store will result in a `429` status error and your additional requests will fail.
```
Comment (slsriehl) [inline reference/price_lists.v3.yml:1773]:
```suggestion
        * Supports up to 50 simultaneous GET requests. Running more than the allowed number of requests concurrently on the same store will result in a `429` status error and your additional requests will fail.
```

File: reference/price_lists.v3.yml [hunk 3]
```diff
@@ -850,7 +850,12 @@ paths:
       tags:
         - Price Lists Records
       summary: Get All Price List Records
-      description: Returns a list of *Price List Records* associated with a *Price List*.
+      description: |-
+        Returns a list of *Price List Records* associated with a *Price List*.
+
+        **Notes**
+        * Up to 10 simultaneous GET requests are supported with this API. Running more than the allowed concurrent requests in parallel on the same store will result in a `429` status error and your additional requests will fail.
+        * In order to maximize performance, it's recommended to store Pricelist Records data to reduce the number of calls.
```
Comment (slsriehl) [inline reference/price_lists.v3.yml:858]:
```suggestion
        * Store Pricelist Records data to reduce the number of calls and maximize performance.
```

**Comment by bc-tgomez** [comment]
@slsriehl I got the checks to pass now, do we still need a creator review?

**Review by slsriehl — APPROVED** [review]

**Pull request merged**